components/Dashboard/Settings/SocialForm.tsx

```diff
@@ -1,83 +1,119 @@
 import React from 'react'
 import { useForm } from 'react-hook-form'
+import { toast } from 'react-toastify'
+import { ApolloError } from '@apollo/client'
+import { toLower } from 'lodash'
 import { useTranslation } from '../../../config/i18n'
 import SettingsForm from '../../../components/Dashboard/Settings/SettingsForm'
 import SettingsFieldset from '../../../components/Dashboard/Settings/SettingsFieldset'
-import XIcon from '../../../components/Icons/XIcon'
 import Button, { ButtonVariant } from '../../../elements/Button'
+import { SocialPlatform, SocialFormField } from './SocialFormField'
+import { SocialMedia, useUpdateSocialMediaMutation } from '../../../generated/graphql'
 import theme from '../../../theme'
+import FormError from '../../FormError'
 
-const SocialForm: React.FC = () => {
+type FormData = {
+  facebook: string
+  instagram: string
+  youtube: string
+  website: string
+  [key: string]: string
```

Comment:
Is expanding FormData to take string->string keys necessary? It seems like the four known types cover all the bases but maybe there's something I'm not seeing

Reply:
I removed it. Previously social media were stored as an array.